i18n/en-US/help/perms.json

```diff
@@ -0,0 +1,40 @@
+{
+    "DESCRIPTION": "Sends a help embed in regards to permissions.",
+    "LEVELS": "Permission Levels",
+    "VIEW_ALL": "There are six possible permission levels to have. Commands for all levels can be seen with `{{prefix}}commands`.",
+    "BLACKLISTED": "» Blacklisted [-1]",
```

Comment:
After uploading the strings to Crowdin, these special characters will not render properly and/or become other characters. Can we use unicode instead? The unicode for `»` is `\u00bb`.